src/templates/package.json

```diff
@@ -26,7 +26,7 @@
 		"@graphql-mesh/soap": "0.14.25",
 		"@graphql-mesh/http": "^0.96.9",
 		"graphql": "^16.6.0",
-		"@adobe/plugin-hooks": "0.3.0",
+		"@adobe/plugin-hooks": "0.3.2-beta",
 		"@adobe/plugin-on-fetch": "^0.1.0",
```

Comment:
@kmaschi Doesn't the version here need to be updated as well to `0.1.1-alpha`?

Reply:
It has been bumped here as well. Thanks for catching this.